packages/fxa-content-server/app/tests/spec/lib/router.js

```diff
@@ -596,7 +595,7 @@ describe('lib/router', () => {
         sinon.stub(router, 'isInReactExperiment').callsFake(() => true);
         routeHandler = router.createReactOrBackboneViewHandler(
           'cannot_create_account',
```

Comment:
I think we will also want to change `'cannot_create_account'` to its view equivalent; passing in `SignInPasswordView` is funky when leaving `cannot_create_account` as-is.